designs/EmbeddedReadmesTechSpec.md

```diff
@@ -0,0 +1,88 @@
+# Embedded Readmes Technical Spec
+
+* Status: Implementing
+* Author(s): [Advay Tandon](https://github.com/advay26)
+* Issue: [6873](https://github.com/NuGet/Home/issues/6873) README.md (markdown description) support for NuGet packages
+
+## Problem Background
+
+Users of NuGet want to be able to include Readme files as part of the package when they `pack` and `push` their project through the commandline. The [design document](https://github.com/NuGet/Home/wiki/Packaging-READMEs-within-the-nupkg) outlines the work needed for this feature and the intended results.
```

Comment:
The link to `https://github.com/NuGet/Home/wiki/Packaging-READMEs-within-the-nupkg` points to a new wiki page. Is it intentional? Are you going to create a new wiki page once this PR is merged?

Reply:
It looks like the name of the document within the wiki was changed at some point. I've updated the link to point to the correct place now. Thanks for pointing this out!